cumulus_library/studies/core/core_templates/core_utils.jinja

```diff
@@ -1,38 +1,138 @@
+{#- A collection of macros for constructing common safe column selects
```

Comment:
This line feels like it relates to some of these util methods but not others - do you want a `schema.jinja` file? (I'm fine with them still living in `utils` but maybe this comment would more naturally live above the block with the safe-select macros)

Reply:
can you be more explicit about this? i don't get the vibe.

Reply:
Oh just that `collection of macros for constructing common safe column` looks to be a file-level description but this file has other general utils stuff like loop comma handling, etc. I was just suggesting moving this comment above the schema-related methods.

Reply:
Did the spirit rather than the letter - i marked the loop comma thing as needing to move to a tbd global macro location.